### ISSUE TITLE
problem compiling your examples

### ISSUE THREAD

**Issue by TonvanWissen** [Links2004/arduinoWebSockets#32]
I'm learning about websockets to use in my project.
Your example :WebSocketServer_LEDcontrol.ino seems to be just what I need! Thank you!
But: trying to compile (win10, Arduino 1.6.7) I keep getting this error:

> Arduino: 1.6.7 (Windows 10), Board:"NodeMCU 0.9 (ESP-12 Module), 80 MHz, Serial, 115200, 4M (3M SPIFFS)"
> C:\Users\Ton\Documents\Arduino\libraries\arduinoWebSockets\src\WebSockets.cpp:28:35: fatal error: core_esp8266_features.h: No such file or directory
 > #include \<core_esp8266_features.h\>
 >                                                              ^
> compilation terminated.
> exit status 1

What can I do? 
Thank you for helping me out ...

Ton

**Comment by greengiant83** [comment]
I just ran into this last night. Its been answered in a few of the closed issues. Basically you just need to install the board from git instead of through the board manager.

**Comment by Links2004** [comment]
yes, the latest git version, needs the git version of the ESP core to work.
#30

**Comment by TonvanWissen** [comment]
Ok, thankyou greengiant and Links2004, for your answer.
I'm very new to the Git.
I'll uninstall the board (through board manager) and install it new from git.
Never done that before, I guess it is just copying the files to the right place?

Thanks again and happy holidays!
Ton

**Comment by Links2004** [comment]
https://github.com/esp8266/Arduino#using-git-version-

**Comment by TonvanWissen** [comment]
Sorry for bothering you all with my beginners problems, but I'm not there yet...

I have copied the complete repositry to C:\\Program Files (x86)\Arduino\hardware\esp8266com\
in a directory called: esp8266.
The board is recognized bij Arduino1.6.7 but still I get an error:

> Arduino: 1.6.7 (Windows 10), Board:"NodeMCU 0.9 (ESP-12 Module), 80 MHz, Serial, 115200, 4M (3M SPIFFS)"

>exec: "C:\\Program Files (x86)\\Arduino\\hardware\\esp8266com\\esp8266/tools/xtensa-lx106-elf/bin/xtensa-lx106-elf-g++": file does not exist.

And indeed I can't find this file on my computer.
Also: the command 'git' is not recognized by the windows cmd. 

Should I have used the Github Desktop to Clone it?

Thank you again for your time ..
